src/databricks/labs/ucx/source_code/source_migrator.py

```diff
@@ -6,15 +6,17 @@
 from databricks.labs.ucx.source_code.dependencies import DependencyGraph, Dependency, DependencyLoader
 
 
-class NotebookMigrator:
+class SourceCodeMigrator:
```

Comment:
i'd not rename this just yet. notebook migrator and source code migrator are two different things, per se. `NotebookMigrator` works on top of workspace client, backing up the notebooks via API, it'll have a lot of logic specific to running within the workspace

Reply:
done